mlir/include/mlir/Dialect/MIOpen/AffineMapHelper.h

```diff
@@ -90,7 +90,15 @@ inline bool hasPadding(AffineMap map) {
   map.walkExprs([&ret](AffineExpr expr) { ret |= hasPadding(expr); });
   return ret;
 }
-
+// check expr is 1 or 0
```

Comment:
it's hard to understand the comment.
you are explaining the implementation detail, but not really what the function does.

Reply:
done